src/XrdAcc/XrdAccPrivs.hh

```diff
@@ -50,6 +50,8 @@ enum XrdAccPrivs {XrdAccPriv_All    = 0x07f,
                   XrdAccPriv_Readdir= 0x020,
                   XrdAccPriv_Write  = 0x040,
                   XrdAccPriv_Update = 0x060,
+                  XrdAccPriv_Stage  = 0x080,  // Stage
```

Comment:
Didn't we decide that "stage' also includes 'poll" as there is no use case without having both of them? If so, this 0x180.